apps/hash-graph/lib/graph/src/knowledge/entity.rs

```diff
@@ -127,7 +127,7 @@ pub struct LinkData {
 #[serde(deny_unknown_fields, rename_all = "camelCase")]
 pub struct EntityMetadata {
     record_id: EntityRecordId,
-    version: EntityVersion,
+    version: EntityTemporalMetadata,
```

Comment:
Do we want to rename `version`?

Reply:
Do you have a suggestion on how we should name it?
- `temporal_metadata`
- `temporal` (we have `provenance` for `ProvenanceMetadata`
- `temporal_version`

Reply:
Applied in [Rename fields in EntityMetadata](https://github.com/hashintel/hash/pull/1854/commits/22de73165691e32a391ec74bce5b2ac2d0a92def)